NeoML/include/NeoML/Dnn/Dnn.h

```diff
@@ -388,6 +388,9 @@ class NEOML_API CBaseLayer : public virtual IObject {
 	// Indicates if the layer performs in-place processing (after the Reshape method call)
 	bool isInPlace;
 
+	// Copies paramBlobs to dist layers
```

Comment:
The comment is not precise, you should write something like:
`Let the 'dist' layer's paramBlobs be the pointers to this layer's paramBlobs data`
but more correct  grammatically ;)